### ISSUE TITLE
What is `dao.get_opt()`? Is README outdated?

### ISSUE THREAD

**Issue by l1048576** [ivanceras/rustorm#31]
I tried to write tiny code but compilation was failed:

```
src/sozai/mod.rs:46:26: 46:48 error: mismatched types:
 expected `uuid::Uuid`,
    found `core::option::Option<&rustorm::dao::Value>`
(expected struct `uuid::Uuid`,
    found enum `core::option::Option`) [E0308]
src/sozai/mod.rs:46             resource_id: dao.get("resource_id"),
                                             ^~~~~~~~~~~~~~~~~~~~~~
```

`dao::Dao` is defined as `BTreeMap<String, Value>`, so `Dao::get()` returns `Option` and `Dao::get_opt` doesn't seem available.

How to use these methods?
Are [these examples](https://github.com/ivanceras/rustorm/blob/3cacde93d5a81ce03835888e6c6edf9e402e2bd1/README.md#example) wrong?

**Comment by ivanceras** [comment]
Hi, I'm sorry if I broke the API, I am currently working on specific project that uses this as a library and haven't spend much time polishing this library. I only cared for the parts where I use. If I may know your specific requirements, maybe what you are looking for an ORM is diesel, as it allows a high level abstraction on your data with high performance code. Rustorm is aimed at a the very low-level area of managing data.
